src/OpenTelemetry.Exporter.Geneva/GenevaLogExporter.cs

```diff
@@ -303,9 +303,6 @@ internal int SerializeLogRecord(LogRecord logRecord)
                     case double vd:
                     case string vs:
                         break;
-                    default:
-                        value = this.convertToJson(value);
-                        break;
                 }
 
```

Comment:
Remove other references of `convertToJson` from this file.

Reply:
updated according to the comments